.docker/docker-compose.yml

```diff
@@ -1,44 +1,50 @@
 version: '3.4'
 services:
-  local-storage-service-database:
-    build: ./database
-    restart: always
+  # File Storage
+  local-storage-database:
+    build: ./localstorage-db
     environment:
       POSTGRES_USER: ${LOCAL_STORAGE_SERVICE_DATABASE_USER}
       POSTGRES_PASSWORD: ${LOCAL_STORAGE_SERVICE_DATABASE_PASSWORD}
       POSTGRES_DB: verapdfdb
     volumes:
         - type: volume
-          source: filestorage_database_data
+          source: localstorage_database_data
           target: /var/lib/postgresql/data
-  local-storage-service:
+    restart: always
+  file-storage:
     build: ../local-storage-service/server
     expose:
       - 8080
     depends_on:
-      - local-storage-service-database
+      - local-storage-database
     environment:
       LOCALSTORAGE_DATABASE_USERNAME: ${LOCAL_STORAGE_SERVICE_DATABASE_USER}
       LOCALSTORAGE_DATABASE_PASSWORD: ${LOCAL_STORAGE_SERVICE_DATABASE_PASSWORD}
-      LOCALSTORAGE_DATABASE_DB: verapdfdb    # the same as in local-storage-service-database
-      LOCALSTORAGE_DATABASE_HOST: local-storage-service-database
+      LOCALSTORAGE_DATABASE_DB: verapdfdb    # the same as in file-storage-database
+      LOCALSTORAGE_DATABASE_HOST: local-storage-database
       LOCALSTORAGE_DATABASE_PORT: 5432
-      LOCALSTORAGE_DISK_MIN_SPACE_THRESHOLD: ${LOCAL_STORAGE_SERVICE_DISK_MIN_SPACE_THRESHOLD}
+      LOCALSTORAGE_DISK_MIN_SPACE_THRESHOLD: ${LOCAL_STORAGE_SERVICE_DISK_MIN_SPACE_THRESHOLD:-5GB}
       LOCALSTORAGE_MAX_FILE_SIZE: 100MB
       LOCALSTORAGE_MAX_REQUEST_SIZE: 101MB
     volumes:
       - type: volume
-        source: filestorage_server_files
+        source: localstorage_server_files
         target: /opt/verapdf/localstorageservice/files
     restart: always
+
+  # Job Service
   job-service:
     build: ../job-service/server
     expose:
       - 8080
     restart: always
 
```

Comment:
Do we need dependency on queue in job-service?

Reply:
Not yet, I expect it to appear in scope of [issue#12: [BE]: Implement job execution](https://github.com/veraPDF/verapdf-webapp-server/issues/12)